src/app/functions/server/onerep.ts

```diff
@@ -439,3 +449,47 @@ export async function fetchAllPages<Data>(
 
   return dataList.flat();
 }
+
+// Local instance map to cache results to prevent excessive API requests
+// Would be nice to share this cache with other pod via Redis in the future
+const profileStatsCache = new Map<string, ProfileStats>();
+export async function getProfilesStats(
```

Comment:
Is this called from somewhere yet?

Reply:
Not yet, but as soon as we get the landing page done, we will